pkg/authorization/apiserver/admission/restrictusers/restrictusers.go

```diff
@@ -167,19 +172,17 @@ func (q *restrictUsersAdmission) Validate(a admission.Attributes) (err error) {
 		return nil
 	}
 
-	// TODO: Cache rolebinding restrictions.
-	roleBindingRestrictionList, err := q.roleBindingRestrictionsGetter.RoleBindingRestrictions(ns).
-		List(metav1.ListOptions{})
+	roleBindingRestrictionList, err := q.rbrLister.List(labels.Everything())
 	if err != nil {
 		return admission.NewForbidden(a, err)
```

Comment:
while you are here, change all of the `return admission.NewForbidden(a, err)` to be something like `return admission.NewForbidden(a, fmt.Errorf("<useful message here>: %v", err))`

Reply:
will do, I'll let tests run w/ this iteration first. : )